packages/cli/src/create-organization.ts

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import configuration  from "./configuration";
+import { writeToConfigFile, fileExists, readFromJsonFile } from "./utils";
+
+
+export async function createOrganization(name: string, username: string, type: string) {
+  let executionError = false;
+  try {
+    if (!(await fileExists(configuration.configFilePath))) {
+      console.log("Spheron config file does not exist");
+      return;
+    }
+    const jwtToken = await readFromJsonFile("jwtToken", configuration.configFilePath);
+    if (!jwtToken) {
+      console.log("JWT token not valid or does not exist. Pleas login first");
```

Comment:
don't mention JWT anywhere
```suggestion
      console.log("For creating a new organisation, you need to login to Spheron first");
```